### DISCUSSION TITLE
Move folder after successful extraction?

### DISCUSSION THREAD

**Discussion by ManuelReschke** [tfeldmann/organize#393] [Q&A] [answered]
Hello,
Is there a filter that monitors when a folder and files have been successfully extracted from an archive? I mean, if the file size no longer changes, that is the case. And then it is supposed to be deleted. Or is that already possible?

**Comment by tfeldmann** [comment]
Can you expand on your usecase? So you extract an archive and after that organize should run?

**Reply by ManuelReschke** [reply]
Yes, exactly. Another process extracts an archive and after this is complete, "organise" should then move the finished folder.
But actually it should monitor whether the file size of a folder changes. Because otherwise it doesn't know when the unpacking is finished. Or am I making a mistake? Is it actually just like a download?

**Reply by tfeldmann** [reply, marked as answer]
This is not something built into organize, but I see two possible ways to solve this:

1. Create a script which first unpacks the file and runs organize afterwards
2. Create a script monitoring the file size (or watch for the other process to finish) and then run organize

**Reply by j-lakeman** [reply]
https://github.com/tfeldmann/organize/discussions/485
